pkg/config/setup/config.go

```diff
@@ -120,6 +120,9 @@ const (
 
 	// DefaultNetworkPathMaxTTL defines the default maximum TTL for traceroute tests
 	DefaultNetworkPathMaxTTL = 30
+
+	// DefaultRegistryFilename is the default registry filename
+	DefaultRegistryFilename = "registry.json"
```

Comment:
Why more this default here if it's never used in the config ? This creates unnecessary dependency to the `setup` package.